js/gated-content/src/scss/components/_paragraph-headline.scss

```diff
@@ -0,0 +1,54 @@
+.paragraph-headline {
+  display:             flex;
+  justify-content:     center;
+
+  background-image:    url('/modules/contrib/openy_gated_content/assets/img/paragraph-headline.png');
```

Comment:
Not sure that this will work for all projects (some of them can be installed to a subdirectory). Can we use a relative path?